### ISSUE TITLE
Fix documentation for windows setup

### ISSUE THREAD

**Issue by otraore** [EngoEngine/engo#319]
Our current README suggests that to get engo up and running you need to install several packages on windows using [win-builds](http://win-builds.org/doku.php/start). Yesterday me and @makufiru both goth engo demos building and running without any of said packages. Also we've found out that none of the packages mentioned are even [available](http://win-builds.org/doku.php/1.5.0_packages) on win-builds. This issue is made to address this and update the README since it is a bit confusing. All those who wish to try building the demos on windows without win-builds are welcomed to further prove or disprove this point. If this is true this will definitely make windows setup much easier and hassle-free.

**Comment by EtienneBruines** [comment]
I'm not sure why it says "download all *above* packages using win-builds". The idea of `win-builds` was to install simply everything they provide, and then it would work. 

You could do without, but then I'd like to know what you had to install instead. And I'm talking about "from a clean install" - so given you do a fresh installation of Windows 7/8/10, install Go right after that. So what would you need, to successfully build? 

You need something, and that something is being provided by `win-builds`. So I'm not sure what packages exactly are required (probably those providing GLX/GLEW/OpenGL bindings). If anyone has a list of those, that's also welcome. 

So basically we cannot do that much until we have a "confirmed list" of things we'd have to install.

**Comment by otraore** [comment]
I think all that's needed is a c(++) compiler. Either mingw or msys or cygwin should do fine if I'm not mistaken. appveyor for example uses mingw.

**Comment by otraore** [comment]
Bump

**Comment by Luukdegram** [comment]
I was able to build and run all demo's on a clean Windows 10 install using [mingw](http://mingw-w64.org/doku.php/start) (All default settings)

**Comment by otraore** [comment]
Thank you for reassuring my theory @Luukdegram . I'm going to submit a PR soon to fix this since this is a big issue that might shy away windows developers (Our current windows setup documentation).

**Comment by paked** [comment]
Closed by #348
